src/routes/__root.tsx

```diff
@@ -1,20 +1,20 @@
 import { createRootRoute, Outlet } from '@tanstack/react-router';
 import React, { Suspense } from 'react';
-import { SideNavigation } from '../components/SideNavigation';
+// import { SideNavigation } from '../components/SideNavigation';
 
 const TanStackRouterDevtools =
   process.env.NODE_ENV === 'production'
     ? () => null
     : React.lazy(() =>
-        import('@tanstack/router-devtools').then(res => ({
-          default: res.TanStackRouterDevtools,
-        }))
-      );
+      import('@tanstack/router-devtools').then(res => ({
+        default: res.TanStackRouterDevtools,
+      }))
+    );
 
 export const Route = createRootRoute({
   component: () => (
     <>
-      <SideNavigation />
+      {/* <SideNavigation /> */}
```

Comment:
is there a reason why it is commented out?

Reply:
I think side navigation will need more work because it is covering my content @sarahgm

Reply:
Now i have set it back, doesn't look as intended to be but anyway it isn't part if this PR @sarahgm